frontends/web/src/routes/buy/info.tsx

```diff
@@ -87,7 +87,7 @@ class BuyInfo extends Component<Props, State> {
         )
         .then(results => results.filter(result => result))
         // @ts-ignore
-        .then(accounts => accounts.map(({ name, code }) => ({ text: name, value: code })))
+        .then(accounts => accounts.map(({ isToken, name, coinName, code }) => ({ text: isToken ? name : name + ' (' + coinName + ')', value: code })))
```

Comment:
or `` `${name} (${coinName})` ``

Reply:
On a fresh setup name and coinName will be the same should we still display coinName?

![Screen Shot 2021-05-20 at 6 35 18 AM](https://user-images.githubusercontent.com/546900/118920070-f59c5280-b935-11eb-882d-8e15432b8946.png)

Reply:
Updated locize dropdown placeholder, from "Select a coin" to "Select an account"

![Screen Shot 2021-05-20 at 6 40 45 AM](https://user-images.githubusercontent.com/546900/118920261-5cba0700-b936-11eb-8bfa-f68c869afaf1.png)

Reply:
I removed the entry altogether - why have almost the same text twice? Pls check the new commit.

Reply:
> On a fresh setup name and coinName will be the same should we still display coinName?

Great catch. Fixed.